scipy/signal/windows.py

```diff
@@ -1658,6 +1736,7 @@ def exponential(M, center=None, tau=1., sym=True):
     ('hamming', 'hamm', 'ham'): (hamming, False),
     ('hanning', 'hann', 'han'): (hann, False),
     ('kaiser', 'ksr'): (kaiser, True),
+    ('lanczos', 'lcz'): (lanczos, True),
```

Comment:
I'd prefer to remove this. These additional spellings are present for backwards compatibility, but they're bad style so we shouldn't add new ones.